<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MD5Calc</class>
 <widget class="QWidget" name="MD5Calc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>110</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MD5Calc</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>461</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QLineEdit" name="password_le"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="password_lbl">
        <property name="text">
         <string>Password</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="md5hash_le">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="md5hash_lbl">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>MD5 Hash</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="clear_btn">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="calc_btn">
        <property name="text">
         <string>Calculate Hash</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
